### ISSUE TITLE
[BUG] Page is reloaded when delete one item

### ISSUE THREAD

**Issue by Fred-Vatin** [avallete/yt-playlists-delete-enhancer#216]
Firefox | OS | SCRIPT
------- | ---- | ---- 
118.0.2 | Win 10 22H2 | 1.6.0
 
**Steps to reproduce**:
Press this button
![image](https://github.com/avallete/yt-playlists-delete-enhancer/assets/22337329/ac279091-4fd2-4568-808d-89e7b73e6de4)

**Expected results**:
Delete item with no reloading.

**Actual results**:
Reload page. Item is deleted.

**Notes**:
This used to work as intented but youtube must have made some changes recently.

**Comment by Fred-Vatin** [comment]
Any chance to get a build with this fix ?

**Comment by avallete** [comment]
> Any chance to get a build with this fix ?

Yup, my bad I though I did it already. The fix is now released under the `1.6.1` 👍
